### DISCUSSION TITLE
PDF Title

### DISCUSSION THREAD

**Discussion by helmiazih** [spatie/browsershot#816] [Q&A]
How do I set a title when viewing a pdf? HTML title tags are not working. Currently, it uses the last URL as the title.

**Comment by noahbazer** [comment]
You'll want to change the meta-data of the pdf in a pdf editor. When viewing a pdf directly, it'll route to the path of that pdf, and as such will use that url.

**Reply by helmiazih** [reply]
I am generate PDF files from HTML content.

use Spatie\LaravelPdf\Facades\Pdf;
 
Pdf::view('assessment.readiness.summary-pdf')
            ->name('readiness_summary.pdf');
